### DISCUSSION TITLE
How to set initial values for complete form ?

### DISCUSSION THREAD

**Discussion by adityapatil9191** [data-driven-forms/react-forms#1470] [Need help] [answered]
How do I set initial values for complete form ? Here is the sample form component code 
https://codesandbox.io/p/sandbox/react-data-driven-autocomplete-forked-9tkfvt?file=%2Fsrc%2Fcomponents%2FaxiosHooks.js
In my case I am getting the following error. Maximum update depth exceed. This can happen when a component repeatedly calls setState inside componetWillUpdate or componenDidUpdate. React limits the number of updates to prevent infinte loops. What am I missing here ?

**Comment by rvsia** [comment, marked as answer]
Hello, there is a https://www.data-driven-forms.org/components/renderer#initialvalues section.
